test/models/validatable_test.rb

```diff
@@ -64,7 +64,7 @@ class ValidatableTest < ActiveSupport::TestCase
     user = create_user
 
     user.password = ''
-    user.password_confirmation = ''
+    user.password_confirmation = 'x'
```

Comment:
That's what I do. Just wanted to offer this fix to save many people trouble doing that.
http://stackoverflow.com/questions/5113248/devise-update-user-without-password/35064640#35064640